state-synchronizer/src/network.rs

```diff
@@ -50,8 +50,8 @@ pub fn add_to_network(
         .add_protocol_handler(
             vec![],
             vec![ProtocolId::StateSynchronizerDirectSend],
-            QueueStyle::FIFO,
-            NETWORK_CHANNEL_SIZE,
+            QueueStyle::LIFO,
+            1,
```

Comment:
maybe this should be its own constant like `NETWORK_CHANNEL_SIZE`? this value seems to be used also for consensus: https://github.com/libra/libra/blob/master/consensus/src/network.rs#L234